src/Generator.php

```diff
@@ -194,6 +194,10 @@ private function allocateLocaleArray($path)
                     continue;
                 }
 
+                $langFiles = config('vue-i18n-generator.langFiles');
+                if(! is_null($langFiles) && ! empty($langFiles) && ! in_array($noExt, $langFiles)) {
```

Comment:
`! is_null($langFiles)` is redundant with the `! empty($langFiles)` check. 

` if(! empty($langFiles) && ! in_array($noExt, $langFiles)) {`

should work the same